internal/cli/opsmanager/clusters/delete.go

```diff
@@ -43,39 +48,162 @@ func (opts *DeleteOpts) Run() error {
 	if !opts.Confirm {
 		return nil
 	}
+
+	err := opts.newHostIds()
```

Comment:
[nit] usually i'd expect a "new" method to return something, this seems more like an `init` kind of.
`init` methods are not great as this means keeping a context, in this case you're keeping it in `opts` where we usually reserve `opts` for the external parameters that users set.
I'd expect this to be more like  `opts.HostIDs` where it returns a slice of ids to pass to another function